calculator/src/App.js

```diff
@@ -1,25 +1,65 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { useState, useEffect } from 'react';
```

Comment:
Nice to seek hooks in use here, but do you feel this was necessary for this particular application?

Reply:
I trying to go 100% hooks. right now I'm working in a short project with hooks and redux and that why I'm maybe overenthusiastic with hooks